src/Microsoft.Azure.WebJobs.Host/JobHostConfiguration.cs

```diff
@@ -123,6 +125,31 @@ public string HostId
             }
         }
 
+        /// <summary>
+        /// Gets the host machine ID. All host instances on the same machine will have the same HostMachineId.
```

Comment:
>same machine [](start = 64, length = 12)

What is the significant of "same machine"?   (this is potentially dangerous - I'm trying to understand the invariants and what assumptions we're making about this value and what we're using this for)  
If 2 instances are on different machines, can they return the same HostMachineId?